src/gui/org/deidentifier/arx/gui/view/impl/menu/EditorCriterionStackelbergPrivacy.java

```diff
@@ -0,0 +1,122 @@
+package org.deidentifier.arx.gui.view.impl.menu;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.deidentifier.arx.gui.model.ModelCriterion;
+import org.deidentifier.arx.gui.model.ModelStackelbergPrivacyCriterion;
+import org.deidentifier.arx.gui.resources.Resources;
+import org.deidentifier.arx.gui.view.SWTUtil;
+import org.eclipse.swt.SWT;
+import org.eclipse.swt.events.SelectionAdapter;
+import org.eclipse.swt.events.SelectionEvent;
+import org.eclipse.swt.layout.GridLayout;
+import org.eclipse.swt.widgets.Composite;
+import org.eclipse.swt.widgets.Label;
+import org.eclipse.swt.widgets.Text;
+
+import de.linearbits.swt.widgets.Knob;
+
+public class EditorCriterionStackelbergPrivacy extends EditorCriterion<ModelStackelbergPrivacyCriterion> {
+	
+	private Text labelAdvCost;
+	private Text labelAdvGain;
+	private Text labelPubBenefit;
+	private Text labelPubLoss;
+	
+	Knob<Double> knobAdvCost;
+	Knob<Double> knobAdvGain;
+	Knob<Double> knobPubBenefit;
+	Knob<Double> knobPubLoss;
+	
+	public EditorCriterionStackelbergPrivacy(final Composite parent, final ModelStackelbergPrivacyCriterion model) {
+		super(parent, model);
+	}
+
+	@Override
+	protected Composite build(Composite parent) {
+        // Create input group
+        Composite group = new Composite(parent, SWT.NONE);
+        group.setLayoutData(SWTUtil.createFillHorizontallyGridData());
+        GridLayout groupInputGridLayout = new GridLayout();
+        groupInputGridLayout.numColumns = 6;

```

Comment:
Criteria editor is set up as two rows of six columns. versus one row for all other models.  When switching between models, the second row isn't automatically rendered, manual resize is necessary.  Need to redraw or change layout.